### ISSUE TITLE
Missing CORS header (API)

### ISSUE THREAD

**Issue by rothsandro** [DecliningLotus/fontsource#758]
### Describe the bug

Accessing the new API `https://api.fontsource.org/v1/fonts` from the browser via JS doesn't work anymore because CORS is not configured:

> Access to fetch at 'https://api.fontsource.org/v1/fonts' from origin 'https://fiddle.jshell.net' has been blocked by CORS policy: No 'Access-Control-Allow-Origin' header is present on the requested resource. If an opaque response serves your needs, set the request's mode to 'no-cors' to fetch the resource with CORS disabled.

The bug was introduced in PR #734, previously this worked fine because `enableCors()` was used: https://github.com/fontsource/fontsource/blob/d1ec83267d3c4600576a731577eda51269d43688/api/src/main.ts#L33

### Steps to Reproduce

1. Open https://jsfiddle.net/pftv7xdo/
2. See the error in the devtools console

### Expected behavior

CORS header should be configured to allow access from any origin

**Comment by ayuhito** [comment]
@rothsandro, can you verify if the issue is resolved? I added a Cloudflare transform rule to add the header to all responses as a quick temporary measure. 

I'll eventually update the code to serve the headers directly once I wrap up another PR. Thanks for letting us know!

**Comment by rothsandro** [comment]
@ayuhito Yes, it works now. Thanks for fixing this so quickly.
